bin/node-sass

```diff
@@ -18,91 +18,127 @@ var Emitter = require('events').EventEmitter,
-    '  node-sass --output-style compressed foobar.scss > foobar.css',
-    '  cat foobar.scss | node-sass --output-style compressed > foobar.css',
-    '',
-    'Example: Watch the sass directory for changes, compile with sourcemaps to the css directory',
-    '  node-sass --watch --recursive --output css',
-    '    --source-map true --source-map-contents sass',
-    '',
-    'Options',
-    '  -w, --watch                Watch a directory or file',
-    '  -r, --recursive            Recursively watch directories or files',
-    '  -o, --output               Output directory',
-    '  -x, --omit-source-map-url  Omit source map URL comment from output',
-    '  -i, --indented-syntax      Treat data from stdin as sass code (versus scss)',
-    '  -q, --quiet                Suppress log output except on error',
-    '  -v, --version              Prints version info',
-    '  --output-style             CSS output style (nested | expanded | compact | compressed)',
-    '  --indent-type              Indent type for output CSS (space | tab)',
-    '  --indent-width             Indent width; number of spaces or tabs (maximum value: 10)',
-    '  --linefeed                 Linefeed style (cr | crlf | lf | lfcr)',
-    '  --source-comments          Include debug info in output',
-    '  --source-map               Emit source map (boolean, or path to output .map file)',
-    '  --source-map-contents      Embed include contents in map',
-    '  --source-map-embed         Embed sourceMappingUrl as data URI',
-    '  --source-map-root          Base path, will be emitted in source-map as is',
-    '  --include-path             Path to look for imported files',
-    '  --follow                   Follow symlinked directories',
-    '  --precision                The amount of precision allowed in decimal numbers',
-    '  --error-bell               Output a bell character on errors',
-    '  --importer                 Path to .js file containing custom importer',
-    '  --functions                Path to .js file containing custom functions',
-    '  --help                     Print usage info'
-  ].join('\n')
-}, {
-  boolean: [
-    'error-bell',
-    'follow',
-    'indented-syntax',
-    'omit-source-map-url',
-    'quiet',
-    'recursive',
-    'source-map-embed',
-    'source-map-contents',
-    'source-comments',
-    'watch'
-  ],
-  string: [
-    'functions',
-    'importer',
-    'include-path',
-    'indent-type',
-    'linefeed',
-    'output',
-    'output-style',
-    'precision',
-    'source-map-root'
-  ],
-  alias: {
-    c: 'source-comments',
-    i: 'indented-syntax',
-    q: 'quiet',
-    o: 'output',
-    r: 'recursive',
-    x: 'omit-source-map-url',
-    v: 'version',
-    w: 'watch'
+  flags: {
+    errorBell: {
+      type: 'boolean',
+    },
+    functions: {
+      type: 'string',
+    },
+    follow: {
+      type: 'boolean',
+    },
+    importer: {
+      type: 'string',
+    },
+    includePath: {
+      type: 'string',
```

Comment:
I'm not sure this is fully equivalent with the below, since I think the API is expecting an array in all cases, which was why it coerced it if it wasn't. Maybe that `isMultiple` forces the same thing

Reply:
meow always returns array for `isMultiple` flags according documentation.
I checked behaviour and removed unnecessary code below as well: https://github.com/sass/node-sass/pull/3125/files/be85ce1818a68e45d4f40672fce5424a918bebd9#diff-66e4eb9929e494460303e4a5e5c4ea4252befaf983cc44bfea286987f0509ef9L285